Source/AudioSettingsPage.cpp

```
@@ -163,32 +163,40 @@ bool ThreadCountDisplay::onMouseEvent(const NomadUI::NUIMouseEvent& event) {
+        m_audioEngine->setTestToneEnabled(playing);
         
-        // If stopping, restore stream logic
-        if (!m_isPlayingTestSound) {
-            // Restore stream
-            if (m_onStreamRestore) m_onStreamRestore();
-        }
+        m_testSoundButton->setText(playing ? "Stop Test Sound" : "Test Sound");
+        
+        // NOTE: No stream restore needed - test tone uses existing audio callback
     });
     addChild(m_testSoundButton);
```

Cppcheck (2.19.0)

[style] The function 'reset' is never used.

(unusedFunction)